specification/resources/resource-manager/Microsoft.Authorization/stable/2025-01-01/policyAssignments.json

```diff
@@ -784,6 +788,10 @@
               {
                 "value": "DoNotEnforce",
                 "description": "The policy effect is not enforced during resource creation or update."
+              },
+              {
+                "value": "Enroll",
+                "description": "The policy effect is not enforced during resource creation or update unless the resource is enrolled to the assignment instance. This mode is not supported assignments referencing policy definitions whose modes are neither 'Indexed' nor 'All', or referencing policy sets that reference such policy definitions."
```

Comment:
Rephase: "The policy effect is not enforced during resource creation or update until the resource or scope of the resource is enrolled to the assignment instance. Enrollment occurs upon deployment of the policy enrollment resource."

We will include the limitation of definition mode within documentation - this makes it easier to update as we expand to include more modes in the future.

Reply:
Done